lib/bitreserve/helpers.rb

```diff
@@ -0,0 +1,23 @@
+module Bitreserve
+  module Helpers
+    def self.underscored_hash(hash)
```

Comment:
By looking at the API reference, I see 5 or more entities where this will be useful.
We could treat each of those fields individually, by I believe this is a more powerful solution, and less error-prone